.gitignore

```diff
@@ -19,6 +20,11 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
+# Also ignore elastic state files
+*.st
 
 # Temporary files for e.g. tests
 /tmp
+data/elasticsearch/nodes/0/indices/kUGeeR-sQ4yp2yX-_ICJpA/0/_state/retention-leases-4.st
+apps/argos_api/.elixir_ls/build/_test/.mix/cached_dot_formatter
+apps/argos_aggregation/.elixir_ls/build/_test/.mix/cached_dot_formatter
```

Comment:
Ich glaube das ist unnötig, du kannst die tests einfach aus dem Hauptverzeichnis heraus starten mit mix test. Bei mir werden gar keine .elixir_ls Ordner in den apps angelegt?

Reply:
Ich versuche immer Sachen, die ich nicht commiten will, direkt auf die ignore Liste zu setzen.